docs/developer_documentation/frontend-development.md

```diff
@@ -0,0 +1,186 @@
+# Guide to developing MOLGENIS Frontend
```

Comment:
Its not part of this issue, if we keep adding stuff the PR is never going to get merged. Its best the have the small PRs